### ISSUE TITLE
AttributeError: module 'imageio' has no attribute 'get_reader'

### ISSUE THREAD

**Issue by Joesauris** [imageio/imageio#306]
I get this error when running both the "imageio usage example" found here [http://imageio.readthedocs.io/en/latest/examples.html#convert-a-movie](url) (third one down). For what its worth, the two usage examples above it work fine. 

I also see this error in another script I hope to use, but I thought the usage example was better to define my issue. 

Any help would be appreciated. I'm on windows 10

**Comment by almarklein** [comment]
That is very strange ... Can you do this. and post the results?
```py
>>> import imageio

>>> import os

>>> imageio
<module 'imageio' from 'c:\\dev\\pylib\\imageio\\imageio\\__init__.py'>

>>> imageio.get_reader
<function get_reader at 0x000001A372A227B8>

>>> os.listdir(os.path.dirname(imageio.__file__))
['core', 'freeze.py', 'plugins', 'README.md', 'resources', 'testing.py', '__init__.py', '__pycache__']

>>> os.listdir(os.path.dirname(imageio.__file__) + '/core')
['fetching.py', 'findlib.py', 'format.py', 'functions.py', 'request.py', 'util.py', '__init__.py', '__pycache__']
```

**Comment by Joesaurus** [comment]
>>> import imageio
>>> import os

>>> imageio
<module 'imageio' from 'C:\\Users\\dudebill\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\imageio\\__init__.py''>

>>> imageio.get_reader
<function get_reader at 0x0989F738>

>>> os.listdir(os.path.dirname(imageio.__file__))
['core', 'freeze.py', 'plugins', 'resources', 'testing.py', '__init__.py', '__pycache__']

>>> os.listdir(os.path.dirname(imageio.__file__) + '/core')
['fetching.py', 'findlib.py', 'format.py', 'functions.py', 'request.py', 'util.py', '__init__.py', '__pycache__']

**Comment by almarklein** [comment]
I took the liberty to surround your code with tripple backqoutes (press edit to see how).

The output looks good. Also `imageio.get_reader` is there. When you ran the example, could it be that you made a directory called "imageio"? Python has this weird thing allowing import of directories that are not packages. Otherwise, can you run the script line by line and post the output?

**Comment by Joesaurus** [comment]
Ok that is interesting. Thanks again for helping with this issue. I'll look to see if I've got the non-package directory thing going on and I'll also post the output of the script. 

Thanks again for helping with this issue. I'll post an update as soon as I get an opportunity (which may be a couple days unfortunately).

**Comment by Joesaurus** [comment]
positive update...

I ran each line as you recommended and based on the error report, found that I didn't have ffmeg.exe installed (didn't realize that was something I needed). After doing that, I got the usage example to work, from the python terminal, which is a good step forward. For some reason if I make a script out of it and try to run it from powershell I get the original error "AttributeError: module 'imageio' has no attribute 'get_reader'" but perhaps thats a seperate issue.
